R/ancombc_prep.R

```diff
@@ -57,7 +57,7 @@ tse_construct = function(data, assay_name, tax_level, phyloseq) {
         }
 
         if (is.null(tax_level)) {
-            tax_levels = mia::taxonomyRanks(tse)
+            tax_levels = union(mia::taxonomyRanks(tse), colnames(SummarizedExperiment::rowData(tse)))
             txt = sprintf(paste0("`tax_level` is not speficified \n",
```

Comment:
same

Reply:
Thank you both for addressing the taxonomy rank issue, I really appreciate it. I seem to have lost track of the conversation. Could you please confirm whether the commit is now ready for merging? I tend to publish a new version of `ANCOMBC` in a week. Thanks!

Reply:
The commit is ready.